include/deal.II/fe/mapping_q_internal.h

```diff
@@ -1116,6 +1116,8 @@ namespace internal
       const CellSimilarity::Similarity cell_similarity,
       const typename dealii::MappingQ<dim, spacedim>::InternalData &data,
       std::vector<Point<spacedim>> &                 quadrature_points,
+      std::vector<DerivativeForm<1, dim, spacedim>> &jacobians,
+      std::vector<DerivativeForm<1, spacedim, dim>> &inverse_jacobians,
```

Comment:
Where is `maybe_update_q_points_Jacobians_and_grads_tensor` used?

Reply:
In `fill_fe_values()` and `do_fill_fe_face_values()`.